### PR TITLE
Add Cloud Spanner support without `xo/dburl`

### PR DESCRIPTION
**Pull request by BIwashi**
Thank you for the great tool!

Since https://github.com/k1LoW/runn/pull/481 depended on an external package called `xo/dburl`, this library would need to be modified to work properly, but this PR has implemented an implementation that does not depend on `xo/dburl`.
If the `xo/dburl` fixes are merged, this PR and fix will no longer be necessary.

reference:
- https://github.com/xo/dburl/pull/26

### CODE REVIEW

File: db.go [hunk 1]
```diff
@@ -49,7 +50,14 @@ type DBResponse struct {
 }
 
 func newDBRunner(name, dsn string) (*dbRunner, error) {
-	db, err := dburl.Open(normalizeDSN(dsn))
+	var db *sql.DB
+	var err error
+	if strings.HasPrefix(dsn, "sp://") {
```
Comment (k1LoW) [inline db.go:55]: Can you support `spanner://` as well as https://github.com/xo/dburl?

**Review by k1LoW — COMMENTED** [review]
@BIwashi Thank you for your contribution!!

I have one request, and I will merge it as soon as it is fixed.

File: db.go (hunk 1 above)
Comment (BIwashi) [reply db.go]: @k1LoW 
I researched and found that `spanner://` is also supported. (`sp://` was the alias that had been set)
So I fixed like this. https://github.com/k1LoW/runn/pull/482/commits/c690fad5e24856df4430b708ab46984ba7c144f2
What do you think?

- [dburl/scheme.go at b1661a2a1fc75d11485c718e112f09ed4778010f · xo/dburl](https://github.com/xo/dburl/blob/b1661a2a1fc75d11485c718e112f09ed4778010f/scheme.go#L95)
- [Go Playground - The Go Programming Language](https://go.dev/play/p/DRI8beVGrWP)

**Review by k1LoW — APPROVED** [review]
Thank you!!

**Pull request merged**